README.md

```diff
@@ -1,2 +1,11 @@
 # hydra-examples
+
 Example files for Hydra.
+
+## Examples
+
+### `declarative.json`
+
+`declarative.json` is a configuration meant to be used with Hydra's declarative
+project feature. It describes jobsets that will be added to the project after
+the special `.jobsets` jobset is successfully evaluated.
```

Comment:
Let's move this bit of the README and its .json document to a nicely named directory, in anticipation of growth down the line.